README.md

```diff
@@ -1,7 +1,17 @@
-# HOW TO USE
+# libiosexec
+An execv shim that allows executing shell scripts with elevated privilages on iOS.
```

Comment:
I see it as relevant; the library allows scripts that need elevated privileges to do actions, but I guess I can word that differently.

Reply:
It is not relevant or even true, it’s identical to `exec` which uses its parents permissions.

Reply:
Unsure what the description of the project is; considering the usage of it, it seems that it "patches" functions that would otherwise not work that require elevated permissions (which is why recent builds of certain packages provided by Procursus don't have a "Operation Not Permitted" message), but I guess I misunderstood. I need to add a better description, and that's open for suggestions since mine is not "even true".

Reply:
The EPERM issue that occurs when passing scripts to execve() is, as far as I can tell, a random error chosen by the iOS Team. It isn't actually a permissions issue, but rather the fact that the iOS kernel does not support running scripts directly.